File: product/conference/src/main/java/ee/ctob/data/Participant.java
package ee.ctob.data;

import ee.ctob.data.enums.Gender;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.sql.Timestamp;
import java.time.Instant;
import java.time.LocalDateTime;
import java.util.Date;
import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "participants")
public class Participant {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Integer id;
    LocalDateTime created;
    @Column(name = "validation_uuid",unique = true)
    UUID validationUUID;
    @Column(name = "participant_uuid",unique = true)
    UUID participantUUID;
    String firstName;
    String lastName;
    String email;
    @Enumerated(EnumType.STRING)
    Gender gender;
    LocalDateTime dateOfBirth;
    String feedback;
}

File: product/conference/src/main/kotlin/ee/ctob/access/data/Participant.kt
//package ee.ctob.access.data
//
//import ee.ctob.api.enums.Gender
//import java.time.LocalDateTime
//import java.util.*
//import javax.persistence.*
//
//@Entity
//@Table(name = "participants", schema = "conference")
//data class Participant(
//    @Id
//    @GeneratedValue(strategy = GenerationType.IDENTITY)
//    val id: Int? = null,
//    val created: LocalDateTime? = null,
//    @Column(name = "validation_uuid", unique = true)
//    val validationUUID: UUID? = null,
//    @Column(name = "participant_uuid", unique = true)
//    val participantUUID: UUID? = null,
//    val firstName: String? = null,
//    val lastName: String? = null,
//    val email: String? = null,
//    @Enumerated(EnumType.STRING)
//    val gender: Gender? = null,
//    val dateOfBirth: LocalDateTime? = null,
//    val feedback: String? = null
//)